src/Backends/SystemUpdate.vala

```diff
@@ -24,6 +24,7 @@ public class SettingsDaemon.Backends.SystemUpdate : Object {
     public struct UpdateDetails {
         string[] packages;
         int size;
+        Pk.Info[] info;
     }
```

Comment:
I wonder if it's better to have a struct like this or just handle over the whole PackageSack? If we want more information for the packages dialog like in https://github.com/elementary/switchboard-plug-about/issues/286 and https://github.com/elementary/switchboard-plug-about/issues/287 it might be easier if we have access to the Pk.Package itself

Reply:
Hmm is this possible via DBus? Also IMHO name, version and kind should be enough?

Reply:
It looks like serializing the PackageSack isn't supported :sweat_smile: